Uniswap/interface
# chore: remove Google Analytics

## Description
Removes Google Analytics as we no longer need to send analytics there.

## Test plan
### QA (ie manual testing)
- Make sure external links continue to work
- Check the content security policy with security

### Automated testing
- [x] Unit test (N/A)

This will be checked off once this PR is merged: https://github.com/Uniswap/interface/pull/6580
